instances.go

```diff
@@ -170,6 +195,36 @@ func (c *Client) SetDualAuthInstancePolicy(ctx context.Context, enable bool) err
 	return err
 }
 
+// SetAllowedIPInstancePolices updates the allowed IP instance policy details associated with an instance.
+// For more information can refet to the Key Protect docs in the link below:
+// <KP link>
+func (c *Client) SetAllowedIPInstancePolicy(ctx context.Context, enable bool, allowedIPs []string) error {
+	policy := InstancePolicy{
+		PolicyType: AllowedIP,
+		PolicyData: PolicyData{
+			Enabled:    &enable,
+			Attributes: &Attributes{},
+		},
+	}
+	if len(allowedIPs) != 0 {
+		policy.PolicyData.Attributes.AllowedIP = allowedIPs
```

Comment:
validate the format of the IPs

Reply:
Some validation from the API side are as follows:
- attributes stuct must be omitted if `enable` == `false`. This implies that you should error if `enable` == `false` and `len(allowedIPs)` != 0
- len(allowedIPs) must be greater than 0 if `enable` == `true`
- all allowedIP items must be in ipv4 or ipv6 networks specified in CIDR format (X.X.X.X/N, X.X.X.X.X.X.X.X/N)

Reply:
Stephanie, I set the condition to verify enable and Length of allowedIPs. I believe the CIDR format of the ips is verified by key protect service. So I would not rather add that check in SDK. I can add it in CLI.

Reply:
I will be updating the document link for allowed ip policy, I see that the required
format for IPs has been mentioned there.